src/bundle/Resources/public/scss/_content-edit.scss

```diff
@@ -1,3 +1,18 @@
+.ez-page-title-content-edit {
```

Comment:
This class lookes strange, why is it in file `content-edit` ? After all base of this class is `page-title-content-edit`